engine/database/testabilities/user_utxo_fixture.go

```diff
@@ -0,0 +1,74 @@
+package testabilities
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/bitcoin-sv/spv-wallet/engine/database"
+	"github.com/bitcoin-sv/spv-wallet/engine/tester/fixtures"
+	"github.com/bitcoin-sv/spv-wallet/models/bsv"
+	"gorm.io/gorm"
+)
+
+var FirstCreatedAt = time.Date(2006, 02, 01, 15, 4, 5, 7, time.UTC)
+
+type userUtxoFixture struct {
+	db                           *gorm.DB
+	t                            testing.TB
+	index                        uint
+	xpubID                       string
+	txID                         string
+	vout                         uint32
+	satoshis                     bsv.Satoshis
+	unlockingScriptEstimatedSize uint64
+}
+
+func newUtxoFixture(t testing.TB, db *gorm.DB, index uint32) *userUtxoFixture {
+	return &userUtxoFixture{
+		t:                            t,
+		db:                           db,
+		index:                        uint(index),
+		xpubID:                       fixtures.Sender.XPubID(),
+		txID:                         txIDTemplated(uint(index)),
+		vout:                         index,
+		satoshis:                     1,
+		unlockingScriptEstimatedSize: 106,
+	}
+}
+
+func txIDTemplated(index uint) string {
+	return fmt.Sprintf("b6ebbe78d93e1b81dd2c9c0c6cd67f9bdf832e9c2bb12a1d57f30c%010d", index)
+}
```

Comment:
maybe, for me it's kinda easier for tracking what is this tx, so maybe it's useful also for setting up grandparents,
I'm just thinking if this wouldn't be useful to start the id with those numbers instead of ending it 🤔

Reply:
Shoul I change the pattern here, or we can leave it as is for now?

Reply:
If it's quick - I will do it now. Otherwise - in the next occasion.